dotnet/src/Connectors/Connectors.OpenAI/Core/ClientCore.AudioToText.cs

```diff
@@ -62,15 +62,22 @@ private static AudioTranscriptionOptions AudioOptionsFromExecutionSettings(OpenA
 
     private static AudioTranscriptionFormat? ConvertResponseFormat(string? responseFormat)
     {
-        return responseFormat switch
+        if (responseFormat is null)
         {
-            "json" => AudioTranscriptionFormat.Simple,
-            "verbose_json" => AudioTranscriptionFormat.Verbose,
-            "vtt" => AudioTranscriptionFormat.Vtt,
-            "srt" => AudioTranscriptionFormat.Srt,
-            null => null,
-            _ => throw new NotSupportedException($"The audio transcription format '{responseFormat}' is not supported."),
+            return null;
+        }
+
+        AudioTranscriptionFormat result;
+        switch (responseFormat)
+        {
+            case "json": result = AudioTranscriptionFormat.Simple; break;
+            case "verbose_json": result = AudioTranscriptionFormat.Verbose; break;
+            case "vtt": result = AudioTranscriptionFormat.Vtt; break;
+            case "srt": result = AudioTranscriptionFormat.Srt; break;
+            default: throw new NotSupportedException($"The audio transcription format '{responseFormat}' is not supported.");
```

Comment:
Is this change required? The same is related to other `switch/case` changes in this PR. If it's not required, let's try to minimize the number of changes needed for package upgrade.

Reply:
Unfortunately this change is required, as their constructor changes breaks the usage of switch expressions `ArgumentNullException`.